ShowdownAI.Middleware/ShowdownAI.Middleware/Models/Pokemon.cs

```diff
@@ -2,20 +2,31 @@
 {
     public class Pokemon
     {
-        public string Ident { get; set; }
-        public string Details { get; set; }
-        public string Condition { get; set; }
+        public string Ability { get; set; }
         public bool Active { get; set; }
-        public Stats Stats { get; set; } 
-        public List<string> Moves { get; set; }
         public string BaseAbility { get; set; }
+        public bool Commanding { get; set; }
+        public string Condition { get; set; }
+        public string Details { get; set; }
+        public string Ident { get; set; }
         public string Item { get; set; }
+        public List<string> Moves { get; set; }
         public string Pokeball { get; set; }
-        public string Ability { get; set; }
-        public bool Commanding { get; set; }
         public bool Reviving { get; set; }
-        public string TeraType { get; set; }
+        public Stats Stats { get; set; } 
+        public Status Status {get ; set;}
         public string Terastallized { get; set; }
+        public string TeraType { get; set; }
```

Comment:
If we're doing enums for things like Status it might be an idea to convert almost everything to an enum. The AI model will use numbers for its input anyway so these would be converted regardless later in the pipeline. Wdyt?

Reply:
I'd normally opt for an enum if we know it's a pre-defined list of options, can be a backlog story.